src/libos/src/time/mod.rs

```diff
@@ -252,3 +252,22 @@ impl TimeProvider for OcclumTimeProvider {
         }
     }
 }
+
+// For Timerfd
+#[repr(C)]
+#[derive(Debug, Default, Copy, Clone)]
+#[allow(non_camel_case_types)]
+
+pub struct itimerspec_t {
+    it_interval: timespec_t,
+    it_value: timespec_t,
+}
+
+impl itimerspec_t {
```

Comment:
use Default tait is enough, please remove the `fn new ...`

Reply:
Done